src/components/logo/logo.component.tsx

```diff
@@ -1,10 +1,26 @@
+import { ConfigObject } from "@openmrs/esm-config";
 import * as React from "react";
 
-const Logo = ({ width = "110", height = "40" }) => {
+interface LogoProps {
+  config: ConfigObject;
+  width?: number;
+  height?: number;
+}
+
+const Logo: React.FC<LogoProps> = ({ width = 110, height = 40, config }) => {
+  const { logo } = config;
   return (
-    <svg role="img" width={width} height={height}>
-      <use xlinkHref="#omrs-logo-partial-grey"></use>
-    </svg>
+    <>
+      {logo?.src ? (
+        <img src={logo.src} alt={logo.alt} width={width} height={height} />
```

Comment:
What if logo.alt is `undefined`, Why not use optional chaining here also?

Reply:
Good suggestion @harshit151001. I guess we should consider displaying a default alternate text eg. "logo" if logo.alt is null/undefined.

Reply:
`logo.alt` will never be undefined. "Logo" is the default provided by the config schema.

Reply:
Oops, overlooked it!